starter/script.js

```diff
@@ -1 +1,191 @@
-// put code here!
+// Any path the user goes down must ask them at least three questions.
+// There must be a minimum of seven total destinations the user could arrive at based on their responses.
+// For at least one of the questions asked, there must be more than two possible user responses.
+// Your code must contain at least one loop.
+// Your code must make use of both string and number user inputs.
+
+
+// Lost In The Wilderness
+
+// set up scoring
+var scoring = {
+	playerScore: 0,
+	playerHealth: 100,
+	computerScore: 0,
+	computerHealth: 0
+};
+
+// create game objects
+var object = {
+	items: {},
+	food: {'unknown berry': {energy: 10}, 'chestnut': {energy: 20}},
+	animals: {'rabbit': {energy: 60}, 'fish': {energy: 45}},
+	weapons: {'stick' : {power: 30}, 'bear spray' : {power: 50}},
+	enemies: {'bear': {health: 100}, 'lone wolf': {health: 70}, 'hunter': {health: 80}}
+};
+
+var playerName = prompt("What is your name?");
+	if (!playerName) {
+		playerName = "Wanderer";
+	}
+
+// want to add stop function on user clicking 'Cancel'
+
+function invalidEntry() {
+	alert("You have to do something.");
+}
+
+	alert("Hello, " + playerName + "\n\nAre you ready for an adventure?");
+	alert("You are alone, lost in the wilderness.\n\nNo one knows you are out here. It has been two days, and food is running scarce.\n\nYou've got to find your way home.");
+
+
+// player's first choice
+function leftOrRight() {
+	var choice = prompt("You've been making your way south since sunrise through the forest. You come to a fork in the trail.\n\nYou can go [L]eft or [R]ight.");
+	if (choice === 'L' || choice === 'l') {
+		riverLeft();
+	} else if (choice === "R" || choice === "r") {
+		riverRight();
+	} else {
+		alert("There is no path here.");
+		leftOrRight();
+	}
+}
+leftOrRight();
+
+function riverLeft() {
+	var choice = prompt("You walk until you see a creek. You follow it downstream until it widens into a small river. You start to feel hungry.\n\nYou can stop to [F]ish, or continue [W]alking.");
+	if (choice === 'W' || choice === 'w') {
```

Comment:
you can write: 
if (choice.toLowerCase === 'w') {

solves the case sensitivity issue.